src/app/models/form.server.model.ts

```diff
@@ -545,6 +545,11 @@ const compileFormModel = (db: Mongoose): IFormModel => {
         required: false,
         default: '',
       },
+
+      newEncryptionBoundary: {
```

Comment:
nit: Would prefer not to use naming that is not futureproofed. What if there's a need to have a second boundary shift 😆 (`newerEncryptionBoundary`)

Understand that this is probably temporary, thus a nit.

Reply:
hmm okay will something like nonE2EEStorageMode instead work? 🤔

Reply:
I usually name them as:
- what it actually is doing `plainResponses` or `unencryptedResponses`
- which project it belongs to `encryptionBoundaryShift` (the next shift can be called `v2`) so that it can contain some domain specific keywords

Reply:
My thought process for `newEncryptionBoundary` was so that it somewhat reads as `newEncryptionBoundary` enabled when you set it as true haha

Reply:
Flag renamed to `encryptionBoundaryShift` in [18e636b](https://github.com/opengovsg/FormSG/pull/6595/commits/18e636b94996008b9da29a6d6cec18af666ded91)